InformationScripting/src/queries/AstQuery.cpp

```diff
@@ -172,15 +171,14 @@ TupleSet AstQuery::toClassNode(QList<TupleSet> input)
 		return false;
 	};
 
-	auto tuples = ts.tuples(canBeInClass);
+	// TODO currently we remove all the converted nodes, this also means we lose all connections:
```

Comment:
Oh, that's an issue. Please think about what's the best thing to do here, but I guess we might need some way of translating these connections. In any case, the previous solution wasn't entire correct, since a link between two methods , i.e. "overrides" would have been translated to a link between two classes, and it wouldn't really make sense in that context.